### ISSUE TITLE
Go through bbr's NONMEM Bayes milestone

### ISSUE THREAD

**Issue by kyleam** [metrumresearchgroup/bbr.bayes#33]
bbr has a NONMEM Bayes milestone with three issues: https://github.com/metrumresearchgroup/bbr/milestone/17

Once the NONMEM Bayes implementation here settles, we should go through those issues.

At a quick glance:

 * metrumresearchgroup/bbr#403 will be covered by the merge of gh-30

 * metrumresearchgroup/bbr#288 may be covered by gh-30, but it's got more discussion to it, so we should check whether any remaining parts should be captured in this tracker/repo

  * metrumresearchgroup/bbr#347 is probably something to take care of with the "audit bbr functions to determine ones that need a tailored S3 implementation for nmbayes" part of gh-30, and that will likely be tracked by a followup issue rather than completed in gh-30.

**Comment by kyleam** [comment]
I'll drop the [NONMEM Bayes initial release](https://github.com/metrumresearchgroup/bbr.bayes/milestone/3) milestone since this doesn't really need to be done before the release.

**Comment by seth127** [comment]
Can we just close this (and the milestone it mentions)? I looked and there aren't any more open issues on [that milestone](https://github.com/metrumresearchgroup/bbr/milestone/17).

**Comment by kyleam** [comment]
@seth127 Yes, you're right, thanks.

For two of the issues above, I noted that there's more discussion in those original issues and we may want to "check whether any remaining parts should be captured in this tracker/repo".  Quickly scanning through those, I'm not spotting anything.

**Comment by kyleam** [comment]
> close this (and the milestone it mentions)

Verified that milestone is already closed.
